structural-variantcalling.wdl

```diff
@@ -43,14 +43,14 @@ workflow SVcalling {
         File referenceFastaDict
         BwaIndex bwaIndex
         String sample
-        String newId = "\'%CHROM\\_%POS\'"
+        Array[String] svtypes = ["DEL", "DUP", "INS", "INV", "BND"]
         Boolean excludeMisHomRef = false
-        Boolean excludeFpDupDel = false
+        Boolean runDupHold = false
         String outputDir = "."
         Map[String, String] dockerImages = {
             "bcftools": "quay.io/biocontainers/bcftools:1.10.2--h4f4756c_2",
-            "clever": "quay.io/biocontainers/clever-toolkit:2.4--py36hcfe0e84_6",
-            "delly": "quay.io/biocontainers/delly:0.8.1--h4037b6b_1",
+            "clever": "biowdl/clever-toolkit:latest",
```

Comment:
Please use a stable tag, not `latest`

Reply:
where can I copy the tag? It only says "latest" https://hub.docker.com/r/biowdl/clever-toolkit/tags?page=1&ordering=last_updated

Reply:
where can I copy the tag? It only says "latest" https://hub.docker.com/r/biowdl/clever-toolkit/tags?page=1&ordering=last_updated